src/commonMain/kotlin/komem/litmus/RunningParams.kt

```diff
@@ -0,0 +1,39 @@
+package komem.litmus
+
+import komem.litmus.barriers.BarrierProducer
+
+data class RunningParams(
```

Comment:
Please rename into `RunOptions`

Reply:
I like Params better :)

Reply:
Then `Parameters` is better I guess (since it is not an abbreviation). 
This is why I prefer `Options` --- this word is shorter and not an abbreviation.

Reply:
Ok, if you prefere "parameters", let's keep them, but I would recommend then to rename to full form `Parameters`.